Sources/PubNub/PubNub.swift

```diff
@@ -974,11 +1002,16 @@ extension PubNub {
     custom requestConfig: RequestConfiguration = RequestConfiguration(),
     completion: ((Result<[String], Error>) -> Void)?
   ) {
-    route(PushRouter(.manageAPNS(pushToken: deviceToken, environment: environment,
-                                 topic: topic, adding: [], removing: []),
-                     configuration: configuration),
-          responseDecoder: RegisteredPushChannelsResponseDecoder(),
-          custom: requestConfig) { result in
+    route(
+      PushRouter(
+        .manageAPNS(
```

Comment:
@CraigLn we are inside of `listAPNSPushChannelRegistrations` and it looks like not really correct to use here `manageAPNS`.
Is it `listPushChannels` which should be used here instead?

Reply:
This is listing APNS2 registrations exclusively.  The server [API REST docs](https://www.pubnub.com/docs/rest-api#push-notifications-apns-http2) seem to indicate it's the same API as Add/Remove:

Reply:
@CraigLn go it. Then it just confusing enum field name for this type of API.

Reply:
The router is an internal concept that is meant to map directly to the PubNub server APIs, so they are not meant to be 100% aligned with the public PubNub instance method names.   A new router enum value would typically only be added when the server adds a new API.